code/modules/hydroponics/seeds.dm

```diff
@@ -1676,7 +1676,7 @@
 	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/vaporsac)
 	plant_dmi = 'icons/obj/hydroponics2.dmi'
 	plant_icon = "vaporsac"
-	chems = list(VAPORSALT = list(1,1))
+	chems = list(VAPORSALT = list(1,2))
```

Comment:
No, the second value is how much to divide by

I know this is intuitive. Why else do you think there are about 3 people who understand plantcode?